### ISSUE TITLE
Checkpoint not found error in the "structured_data_classification" example

### ISSUE THREAD

**Issue by chenmin1968** [keras-team/autokeras#1214]
### Bug Reproduction
Code for reproducing the bug:

train_file_path = "data/train.csv"
test_file_path = "data/eval.csv"

x_train_df = pd.read_csv(train_file_path)
print(type(x_train_df)) # pandas.DataFrame
y_train_df = x_train_df.pop('survived')
print(type(y_train_df)) # pandas.Series

# Preparing testing data.
x_test_df = pd.read_csv(test_file_path)
y_test_df = x_test_df.pop('survived')

# It tries 3 different models.
clf = ak.StructuredDataClassifier(
    overwrite=True,
    max_trials=3,
    seed=66,
    project_name='song4',
    directory='akeras_models/'
)
# Feed the structured data classifier with training data.
clf.fit(x_train_df, y_train_df) 

# Predict with the best model.
predicted_y = clf.predict(x_test_df)
# Evaluate the best model with testing data.
print(clf.evaluate(x_test_df, y_test_df))
Data used by the code:

### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu 18.04
 - Python: 3.6
 - autokeras: 1.0.3
 - keras-tuner: '1.0.2rc0'
 - scikit-learn: 0.22.1
 - numpy:
 - pandas:
 - tensorflow: 2.2.0
